Corona-Warn-App/src/main/java/de/rki/coronawarnapp/nearby/modules/diagnosiskeyprovider/DefaultDiagnosisKeyProvider.kt

```diff
@@ -34,8 +34,9 @@ class DefaultDiagnosisKeyProvider @Inject constructor(
         }
 
         if (!submissionQuota.consumeQuota(1)) {
-            Timber.w("No key files submitted because not enough quota available.")
-            return false
+            Timber.e("No key files submitted because not enough quota available.")
```

Comment:
where do we keep track of this, besides the back of our minds?

Reply:
The error? That's just for the log, it's not used, but when it will be used, we already know the correct quota.
If you mean the current state `ENFClientLocalData`